rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,222 @@
+
+Persistent volumes are a very important part of a stateful application. Rancher makes working with volumes in multi-host environments a simple process.
+
+#### Terminology
+
+A `volume plugin` and `volume driver` are used interchangeably in Docker and Rancher documentation. They refer to the same thing: a plugin that provides local or shared persistent volumes to a Docker container.
+
+A `storage driver` pertains to how containers and images are stored and managed on your Docker hosts. Examples are: aufs, btrfs, zfs, devicemapper. These drivers are beyond the scope of Rancher. More information is available [here](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/).
+
+### Managing Volumes
+
+In this section you’re going to learn how to create persistent volumes that may be shared between containers.
+
+#### Create a Volume
+
+You can create a volume with the `rancher volume create` command.
+
+```
+$ rancher volume create --driver local app-data
+```
+
+This will create a new local volume with name `app-data`. The name must start with an alphanumeric character, followed by `a-z0-9`, `_` (underscore), `.` (period) or `-` (hyphen). 
+
+The `--driver` flag specifies the volume driver to use. Docker ships with a `local` volume driver. Volumes using this driver persist their data to the host filesystem and are accessible to any container on the same host.
+
+#### List Volumes
+
+You can list volumes in an environment.
+
+```
+$ rancher volume ls
+```
+
+If you created the `app-data` volume from the previous section, you may wonder why it wasn't listed. You can include `inactive` volumes with the `--all` or `-a` flag.
+
+```
+$ rancher volume ls --all
+```
+
+#### Delete a Volume
+
+You can delete a volume with the `rancher volume rm` command.
+
+```
+$ rancher volume rm app-data
+```
+
+#### Volume States
+
+Volumes have five distinct states: `inactive`, `activating`, `active`, `detached`, and `removed`.
+
+A newly-created volume is in `inactive` state until a container attempts to attach to it.
+
+When a container is created, attached volumes enter `activating` state. Once the container enters the `running` phase, its volumes enter `active` state. Containers attaching to an already `active` volume will not impact that volume's state.
+
+When all containers attached to a volume are deleted, the volume enters `detached` state.
+
+A deleted volume is in `removed` state. These volumes no longer exist on any Docker host and won't appear in when listing volumes. The volume "shadow" objects exist in Rancher API for audit purposes.
+
+### Volume Scopes
+
+There are two different volume scopes. The scope refers to the level at which the volume is managed by Rancher.
+
+#### Environment Scope
+
+An `environment scope`, or `external` volume may be shared by all containers in an environment. Rancher schedules containers appropriately based on the volume driver's access mode. A user doesn't need to be aware of the technical
```

Comment:
Environment scope and external are not really the same and I think that we should make that distinction so that we don't paint ourselves into a corner later. Can you drop the "or external" bit?